azure-pipelines/azure-templates/helm-template.yml

```diff
@@ -0,0 +1,46 @@
+
+parameters:
+  - name: 'containerName'
+    type: string
+  - name: 'imageName'
+    type: string
+  - name: 'tag'
+    type: string
+  - name: 'activeSvc'
+    type: string
+  - name: 'aksConnection'
+    type: string
+  - name: 'releaseName'
+    type: string
+  - name: 'nameSpace'
+    type: string
+
+
+
+  
+ 

```

Comment:
why so many random newlines in this & other files? it's sloppy. not blocking.

Reply:
Removed.